aod-discord-bot.js

```diff
@@ -383,22 +395,14 @@ function getPermissionLevelForMember(member) {
 	if (member.permissions.bitfield & BigInt(0x00000008))
 		perm = PERM_OWNER;
 	else {
-		//FIXME very inefficient
-		if (member.roles.cache.find(r => config.adminRoles.includes(r.name)))
-			perm = PERM_ADMIN;
-		else if (member.roles.cache.find(r => config.staffRoles.includes(r.name)))
-			perm = PERM_STAFF;
-		else if (member.roles.cache.find(r => config.divisionCommandRoles.includes(r.name)))
-			perm = PERM_DIVISION_COMMANDER;
-		else if (member.roles.cache.find(r => config.modRoles.includes(r.name)))
-			perm = PERM_MOD;
-		else if (member.roles.cache.find(r => r.name.endsWith('Officer')) ||
```

Comment:
I think not having this check will probably break some things. We don't update the bot config every time we add a division, we just check for the officer suffix. Also, can this be changed to use config.discordOfficerSuffix instead of "Officer"